[WFLY-19116]: Experimental - Provide a cool boot banner.

ehsavoie opened the pull request:
* Add support for a cool ASCII art banner via a banner.txt.

Jira: https://issues.redhat.com/browse/WFLY-19116
      https://issues.redhat.com/browse/WFCORE-4917

It requires https://github.com/wildfly/wildfly-core/pull/5821

bstansberry commented on ee-feature-pack/galleon-local/src/main/resources/packages/banner/content/bin/banner.txt:
```diff
@@ -0,0 +1,7 @@
+| | /| / / / / __  / /_  / / / / /  / /_  / / / / / /  / __/ | |/_/ __ \/ _ \/ ___/ / __ `__ \/ _ \/ __ \/ __/ __ `/ / 
+| |/ |/ / / / /_/ / __/ / / /_/ /  / __/ / /_/ / / /  / /____>  </ /_/ /  __/ /  / / / / / / /  __/ / / / /_/ /_/ / /  
+|__/|__/_/_/\__,_/_/   /_/\__, /  /_/    \__,_/_/_/  /_____/_/|_/ .___/\___/_/  /_/_/ /_/ /_/\___/_/ /_/\__/\__,_/_/   
+                         /____/                                /_/                                                     
```
WildFly Full isn't really a thing.

bstansberry reviewed (changes requested):
@ehsavoie This won't work correctly for WildFly Preview, as all of the wiring is defined in ee-feature-pack/galleon-local, which is not used in building the wildfly-preview feature pack.

YMMV but I think:

1) The configs/standalone/model change needs to be applied in preview as well
2) The packages/banner stuff should go to ee-feature-pack/galleon-shared, EXCEPT the banner.txt file which stays where it is.
3) There should be a banner.txt in preview/galleon-local as well, but one that says "WildFly Preview". WFP does exist; WildFly Experimental does not.

ehsavoie committed:
[WFLY-19116]: Experimental - Provide a cool boot banner.

 * Add support for a cool ASCII art banner via a banner.txt.

Jira: https://issues.redhat.com/browse/WFLY-19116
      https://issues.redhat.com/browse/WFCORE-4917

Signed-off-by: Emmanuel Hugonnet <ehugonne@redhat.com>

ehsavoie commented:
@bstansberry I hope to have addressed all your concerns :)
